resources/vocabulary.md

```diff
@@ -2,5 +2,115 @@
 
 ## "Basic" concepts
 
+### Things that most programming languages can do
+
+#### Arithmetic operations
+This involves typing expressions like `1 + 2` and getting the answer `3` as an output. The operators `+`, `-`, `*` and `/` are common (universal?), while programming languages also tend to have other operators like `**` (exponential power, e.g. `2 ** 3` returns `8`) or `%` (modulo, e.g. `14 % 4` returns `2`).
+
+#### 'Simple' data types
+These are different kinds of values that can be used while programming. Languages have different types of values, but most have a variety of the following:
```

Comment:
Note that some languages make you say which of these things are, whereas others (like Python) just guess unless you choose to tell them.